qiskit/optimization/algorithms/minimum_eigen_optimizer.py

```diff
@@ -112,10 +113,24 @@ def __init__(self, min_eigen_solver: MinimumEigensolver, penalty: Optional[float
         Args:
             min_eigen_solver: The eigen solver to find the ground state of the Hamiltonian.
             penalty: The penalty factor to be used, or ``None`` for applying a default logic.
+            converters: The converters to use for converting a problem into a different form.
+                If not specified, ``QuadraticProgramToQubo`` will be used.
+
+        Raises:
+            TypeError: When there one of converters is an invalid type.
         """
         self._min_eigen_solver = min_eigen_solver
         self._penalty = penalty
-        self._qubo_converter = QuadraticProgramToQubo()
+
+        if converters is None:
+            self._converters = [QuadraticProgramToQubo()]  # type: ignore
+        elif isinstance(converters, QuadraticProgramConverter):
+            self._converters = [converters]  # type: ignore
+        elif isinstance(converters, list) and \
+                all(isinstance(converter, QuadraticProgramConverter) for converter in converters):
+            self._converters = converters  # type: ignore
+        else:
+            raise TypeError('There are the unsupported types of converters in `converters`')
```

Comment:
Since this code seems to be replicated in several places would it be better to have a common method , say in the optimizer base class, that can be called upon to do this conversion?